.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
What would it matter is someone wasn't using phpStorm? It just wouldn't do anything. What if someone was using phpStorm and didn't have this in their global gitignore? 

I don't see any harm having it here.